<ui version="4.0" >
 <class>CompletedPopup</class>
 <widget class="QWidget" name="CompletedPopup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>71</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameBase" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet" >
    <string/>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QFrame" name="frameHeader" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>0</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>QFrame{background-color:#ffecbf}</string>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="lblTitle" >
     <property name="geometry" >
      <rect>
       <x>24</x>
       <y>2</y>
       <width>228</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>&lt;b>Download completed&lt;/b></string>
     </property>
    </widget>
    <widget class="QToolButton" name="spbClose" >
     <property name="geometry" >
      <rect>
       <x>257</x>
       <y>2</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Close</string>
     </property>
     <property name="text" >
      <string>X</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/buttons/images/cross.png</iconset>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>4</x>
       <y>4</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources/resources.qrc" >:/header/images/tick.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameInfo" >
    <property name="geometry" >
     <rect>
      <x>1</x>
      <y>25</y>
      <width>278</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet" >
     <string>QFrame{background-color:#fff7dd}</string>
    </property>
    <property name="frameShape" >
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="imgVideoService" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>2</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>?</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblVideoTitle" >
     <property name="geometry" >
      <rect>
       <x>40</x>
       <y>2</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>247</green>
           <blue>221</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>127</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>247</green>
           <blue>221</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>117</red>
           <green>116</green>
           <blue>118</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>247</green>
           <blue>221</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="spbPlayVideo" >
     <property name="geometry" >
      <rect>
       <x>240</x>
       <y>7</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip" >
      <string>Play video</string>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >:/buttons/images/play_video.png</iconset>
     </property>
     <property name="iconSize" >
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
